deprecation.py

```diff
@@ -80,7 +85,6 @@ def __str__(self):
 
 class UnsupportedWarning(DeprecatedWarning):
     """A warning class for methods to be removed
-
```

Comment:
I don't think you meant to delete this line?

Reply:
No, I'll fix it

Reply:
Fixed